index.html

```diff
@@ -0,0 +1,911 @@
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          Working From Home, Celebrities Grapple With Revealing Too Much
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          How to Cook and Freeze a Large Piece of Meat, and Eat for Weeks
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          What if You Don’t Want to Go Back to the Office?
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          3 Hospital Workers Gave Out Masks. Weeks Later, They All Were Dead.
+        </p>
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">More in Space and Astronomy</a></h5>
+      <aside class="SpaceAstronomy rows-asside">
+        
+
+        <div class="article-item">
+          <img src="assets/images/one.jpg" alt="" />
+          <p class="figure-caption-items">NASA/GSFC/SDO</p>
+          <p>There’s Something Special About the Sun: It’s a Bit Boring</p>
+          <span class="smallcaption">April 30</span>
+        </div>
+
+        <div class="article-item">
+          <img src="assets/images/two.jpg" alt="" />
+          <p class="figure-caption-items">Jude Guidry/NASA</p>
+          <p>
+            NASA Picks Moon Lander Designs by Elon Musk and Jeff Bezos Rocket
+            Companies
+          </p>
+          <span class="smallcaption">April 30</span>
+        </div>
+
+        <div class="article-item">
+          <img src="assets/images/tree.jpg" alt="" />
+          <p class="figure-caption-items">Namouni and Morais, NASA</p>
+          <p>A Nest of Alien Asteroids Orbits Our Sun</p>
+          <span class="smallcaption">April 28</span>
+        </div>
+      </aside>
+
+      <aside class="SpaceAstronomy rows-asside">  
+
+        <div class="article-item">
+          <img src="assets/images/space.jpg" alt="" />
+          <p class="figure-caption-items">NASA, ESA and STScI</p>
+          <p>Hubble Marks 30 Years of Seeing a Universe Being Born and Dying</p>
+          <span class="smallcaption">April 24</span>
+        </div>
+
+        <div class="article-item">
+          <img src="assets/images/eye.jpg" alt="" />
+          <p class="figure-caption-items">NASA</p>
+          <p>ESA and P. Kalas/University of California, Berkeley
+          The Case of the Disappearing Exoplanet</p>
+          <span class="smallcaption">April 20</span>
+        </div>
+
+        <div class="article-item">
+          <img src="assets/images/BigBang.jpg" alt="" />
+          <p class="figure-caption-items">Kamioka Observatory</p>
+          <p>
+            Institute for Cosmic Ray Research, University of
+            Tokyo.Why the Big Bang Produced Something Rather Than Nothing</p>
+          <span class="smallcaption">April 27</span>
+        </div>
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">Editors’ Picks</a></h5>
+      <aside class="EditorPicks rows-asside">
```

Comment:
In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))